Source/WebKit/UIProcess/Cocoa/SystemPreviewControllerCocoa.mm

```diff
@@ -262,6 +262,7 @@ @interface _WKSystemPreviewDataTaskDelegate : NSObject <_WKDataTaskDelegate> {
     RetainPtr<NSMutableData> _data;
     FileSystem::PlatformFileHandle _fileHandle;
```

Comment:
instead of adding a `_started` boolean can't we initialize `_fileHandle` to `FileSystem::invalidPlatformFileHandle`?

Reply:
yes, thanks!!